rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6551,6 +6552,312 @@ private void assertRefundInProcessPegInVersionLegacy(
         Assert.assertFalse(provider.getHeightIfBtcTxhashIsAlreadyProcessed(btcTx.getHash()).isPresent());
     }
 
+    private Address getFastBridgeAddressFromRedeemScript(Script redeemScript, Sha256Hash derivationArgumentHash) {
+        Script fastBridgeRedeemScript = FastBridgeRedeemScriptParser.createMultiSigFastBridgeRedeemScript(
+            redeemScript,
+            derivationArgumentHash
+        );
+
+        Script fastBridgeP2SH = ScriptBuilder.createP2SHOutputScript(fastBridgeRedeemScript);
+        return Address.fromP2SHScript(bridgeConstants.getBtcParams(), fastBridgeP2SH);
+    }
+
+    private BigInteger registerFastBridgeBtcTransaction_RSKIP293(
+        ActivationConfig.ForBlock activations,
+        Coin valueToSend,
+        boolean includeActiveFederation,
+        boolean includeRetiringFederation,
+        boolean retiringFederationExists
```

Comment:
Don't these two mean the same thing?

Reply:
No. The first one is used to include an output sending value to an address that could belong to a current retiring federation(if there is one) in a BTC TX. And the `retiringFederationExists` parameter is used to set a current retiring federation. So, the outputs sent to that current retiring federation in a BTC TX can be taken into account.